ParselyTrackerTests/MetadataTests.swift

```diff
@@ -39,12 +39,13 @@ class MetadataTests: ParselyTestCase {
             duration: expected["duration"] as? TimeInterval
         )
         let actual: Dictionary<String, Any> = metasUnderTest.toDict()
+        let pubDateUnix: String = String(format:"%i", (expected["pub_date"]! as! Date).timeIntervalSince1970 * 1000)
```

Comment:
Given that the bug report specifically mentioned the `NSDate` type, and given this brief summary of the potential differences, perhaps we should specifically write a test against the Obj-C types as well. There are few differences between the two, but they are there.

https://stackoverflow.com/questions/39811352/swift-3-date-vs-nsdate

Reply:
Looking at this again, the mention of `NSDate` in the customer's report is a bit misleading. It's not possible to initialize a `ParselyMetadata` object with an `NSDate`, since its arguments are typed. The traceback linked [here](https://github.com/Parsely/web/issues/9712#issuecomment-537560971) is functionally indentical to the one sent by the customer, so I feel very confident that this issue is fixed and tested fully.